Sources/Config.php

```diff
@@ -682,7 +682,7 @@ class Config
 				 * Path to the cache directory for the file-based cache system.
 				 */
 				END,
-			'default' => 'dirname(__FILE__) . \'/cache\'',
+			'default' => '__DIR__ . \'/cache\'',
```

Comment:
This is acceptable, but only if the same change is made in ./other/Settings.php.

Reply:
All references to dirname(__FILE__) changed to __DIR__ in Settings.php & Settings_bak.php.